frontend/src/components/Navigation/_Footer.scss

```diff
@@ -0,0 +1,90 @@
+@use "../../components/Basics/" as *;
```

Comment:
Basics and Navigation both belongs in components, so there is no need to go back a second time.